documentation/src/test/java/org/hibernate/userguide/fetching/GraphFetchingTest.java

```diff
@@ -89,7 +85,7 @@ public void test() {
 				Employee.class,
 				userId,
 				Collections.singletonMap(
-					"javax.persistence.fetchgraph",
+					GraphSemantic.FETCH.getJpaHintName(),
```

Comment:
Since this is an example in documentation, I think it would be clearer to just use the string (as it was)?

If you really want to use a constant in documentation, we should also show the import statement, or use the fully qualified classname.

Reply:
Thanks. Didn't realize that. Reverted back the changes to the class. Thanks.